tests/fixtures/ERC20.cairo

```diff
@@ -6,7 +6,7 @@
 from starkware.cairo.common.cairo_builtins import HashBuiltin
 from starkware.cairo.common.uint256 import Uint256
 
-from openzeppelin.token.erc20.library import ERC20
+from tests.fixtures.lib.erc20 import ERC20
```

Comment:
why this change?

Reply:
the openzepplin contracts have an [assert](https://github.com/OpenZeppelin/cairo-contracts/blob/4dd04250c55ae8a5bbcb72663c989bb204e8d998/src/openzeppelin/token/erc20/library.cairo#L188), which reverts if the transfer is being done to the 0 address, in our test runner, what I can see is regardless of whatever I send the caller_address, inside the test it was 0, hence this was reverting, so I have changed the library to disable it.

Reply:
why would we mint to address `0`? I think that it's not necessary

Reply:
resolved, I think probably some human error on my end because of which I got confused.